R/derive_params_growth_age.R

```diff
@@ -366,18 +344,15 @@ derive_params_growth_age <- function(dataset,
         select(-c(P95, Sigma))
     } else {
       add_pctl <- add_pctl %>%
-        mutate(AVAL = pnorm(AVAL) * 100)
+        mutate(AVAL = pnorm(temp_z) * 100)
```

Comment:
@Fanny-Gautier I thought it was this piece that did the trick but let me double check again